packages/logo/src/Logo.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * A logo for Jetpack.
+ *
+ * @package jetpack-logo
+ */
+
+namespace Jetpack\Assets;
+
+/**
+ * Create and render a Jetpack logo.
+ */
+class Logo {
+	/**
+	 * Absolute URL of the Jetpack logo.
+	 *
+	 * @var string
+	 */
+	private $url;
+
+	/**
+	 * Constructor.
+	 * You can optionally pass a URL to override the default one.
+	 *
+	 * @param string $url New URL of the Jetpack logo.
+	 */
+	public function __construct( $url = '' ) {
+		if ( ! $url ) {
+			$url = plugins_url( 'assets/logo.svg', __DIR__ );
+		}
+
+		$this->url = $url;
+	}
+
+	/**
+	 * Build and retrieve an <img /> tag with the Jetpack logo.
+	 *
+	 * @return string The Jetpack logo.
+	 */
+	public function get_image() {
+		return sprintf(
+			'<img src="%s" class="jetpack-logo" alt="%s" />',
+			esc_url( $this->url ),
+			esc_attr__(
+				'Jetpack is a free plugin that utilizes powerful WordPress.com servers to enhance your site and simplify managing it',
```

Comment:
```suggestion
				'Jetpack',
```

Reply:
Is there a different attribute we could use to put the old string as extra infortmation?

Reply:
Yeah, I'd very much like keeping the original string somehow, while making everything accessible. Could we use an aria attribute to sort that out @aldavigdis?

Reply:
I think this is more of a design issue. I suggest using a hyperlink wrapping the image tag somehow like this:

```html
<a
  href="https://www.jetpack.com/"
  rel="external"
  title="Visit the Jetpack website. Jetpack is a free plugin that utilizes powerful WordPress.com servers to enhance your site and simplify managing it."
  >
    <img src="foo.svg" alt="Jetpack." />
</a>
```

Notice that I added a reference to what the link does and a period after the end of the title and alt tags. This makes screen readers pause slightly.

Reply:
Since I prefer not making the logo a link unless we want to, I've gone with the other suggestion - just changed the `alt` attribute to `Jetpack.`.

I'll ask for a design review because of this change too, just to be sure we're good with it.

Reply:
Ah, I've also removed the localization function there, because we no longer need it - we don't translate Jetpack AFAIK.

Reply:
I'd still wrap it with a `__` function, just in case.

Reply:
I'm happy to do it if @Automattic/jetpack-crew are good with it - last time I did it, someone told me we aren't supposed to do it. Hence, I didn't do it here. 

If you ask me, it's unnecessary - we don't localize our plugin name, and that should be a good enough reason not to localize this string.